data/templated/example.xml

```diff
@@ -106,9 +106,9 @@
          <value>1</value>
          <value>1</value>
          <value>1</value>
-         <value dist="uniform" min="1674477220000" max="1706013220000"/>
+         <value dist="uniform" min="1999-12-12 01:01:55" max="1706013220000"/>
```

Comment:
FWIW, this example is a little weird.  Are mixed min/max string vs. int specification types allowed?  I suppose so, but might make more sense to provide two separate examples:
- one with both as ints
- one with both as strings

Reply:
Switched this to string only. Example with int/long already exists